main_logic/tts_client.py

```
@@ -990,3 +1007,241 @@ def get_tts_worker(core_api_type='qwen', has_custom_voice=False):
+    #     return
+    # === 新增：定义断句标点 ===
+    PUNCTUATIONS = {"。", "！", "？", "…", "\n", ".", "!", "?", "；", ";"}
+
+    async def async_worker():
+        # 检查问题使用的代码 --------------
+        logger.info("✅ [LocalTTS] 连接成功")
+        response_queue.put(("__ready__", True))
+        text_buffer = "" # 文本缓冲
+
+        # === 调试点1 确认进入主循环
+        print(">>> [DEBUG] 进入 Worker 主循环，等待文本...")
```

Ruff (0.14.10)

String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)